packages/mgt-chat/src/statefulClient/GraphNotificationUserClient.ts

```diff
@@ -259,22 +259,29 @@ export class GraphNotificationUserClient {
 
       if (!subscription) {
         try {
-          subscription = await this.createSubscription(this.currentUserId);
+          subscription = await this.createSubscription(currentUserId);
         } catch (e) {
           subscription = undefined;
           isRenewalInError = true;
           error(e);
 
-          // rather than 3 seconds, back off to 10 seconds if we get a 403
-          if ((e as { statusCode?: number }).statusCode === 403) {
-            nextRenewalTimeInMs = 10 * 1000;
+          const err = e as { statusCode?: number; message: string };
+          if (err.statusCode === 403) {
+            // rather than 3 seconds, back off to 10 seconds if we get a "403" which should really be a 429 - this happens when we reached subscription limit of 10
+            if (err.message.indexOf("has reached its limit of '10'") > 0) {
```

Comment:
I would just do `has reached its limit` because the limit could change over time.